### PR TITLE
Launch game

### PR DESCRIPTION
**Pull request by sremedios**
# Summary of changes
Can now launch games as desired, specified by `Cargo.toml`
Ignores command line arguments for now, though these can be implemented easily later in `router.rs`

# How was this tested?
Successfully launched steam game BBTAG.

# TODO
Game paths are currently absolute.  This is probably desired in the final product though.

### CODE REVIEW

File: arclightning_backend/System/_App.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[System]
```
Comment (starfys) [inline arclightning_backend/System/_App.ini]: what is this

File: arclightning_backend/src/router.rs [hunk 2]
```diff
@@ -375,7 +383,7 @@ impl Router {
         );
 
         if requested_path == root {
-            *request.uri_mut() = hyper::Uri::from_static("/index.html");
+            *request.uri_mut() = hyper::Uri::from_static("/start.html");
```
Comment (starfys) [inline arclightning_backend/src/router.rs]: i didn't even have to ask, you just did this. gg

Comment (sremedios) [reply arclightning_backend/src/router.rs]: gg

File: arclightning_backend/arclightning_controller.cfg [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿[General]
```
Comment (starfys) [inline arclightning_backend/arclightning_controller.cfg]: y tho

Comment (sremedios) [reply arclightning_backend/arclightning_controller.cfg]: don't we want to allow for the same fight stick mappings to keyboard across devices?

**Pull request merged**